src/ImageService/Startup.cs

```diff
@@ -69,6 +76,11 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddHttpContextAccessor();
 
+            services.AddTransient<IGetImageMessageCommand, GetImageMessageCommand>();
```

Comment:
you don't want to use auto inject?

Reply:
i want, but don't know how to do it yet

Reply:
use this attribute for public interface of class which you want to inject